applications/daos.md

```diff
@@ -0,0 +1,135 @@
+* wetalice Graduated from top5 universities in China, good at economics and game theory. Currently he is the lead developer for the KICO team.
+* Zifan Zhang has worked for NetEase and did front-end development for 6 years. Currently, he has his own entrepreneurial team in Singapore.
+* You Lam is test engineer who has been working in the industry for 9 years.
+### Contact
+* **Contact Name:** JimYam
+* **Contact Email:** transxask1@gmail.com
+### Legal Structure
+* **Registered Address:** N/A
+* **Registered Legal Entity:** N/A
+### Team Code Repos
+
+- [https://github.com/daos-org/daos.git](https://github.com/daos-org/daos.git)
+- [https://github.com/DICO-TEAM/dico-chain.git](https://github.com/DICO-TEAM/dico-chain.git)
+- [https://github.com/listenofficial/listen-parachain.git](https://github.com/listenofficial/listen-parachain.git)
+
+### Development Status
+
+- [https://github.com/daos-org/daos.git](https://github.com/daos-org/daos.git)
+
+***
+
+## Development Roadmap
+### Overview
+* **Total Estimated Duration:** 2 months
+* **Full-Time Equivalent (FTE):** 6FTE
+* **Total Costs:** 10k USD
+### Milestone 1 — Implement Create-Dao, Agency, DoAS, Square, and Sudo Modules
+* **Estimated duration:** 1 month
+* **FTE:** 6
+* **Costs:** 6000 USD
+
+|Number   | Deliverable       | Specification                                                                                                                                                               |
+|-----:  |-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|0a.     | License           | MIT, APACHE2.0                                                                                                                                                              |
+|0b.     | Documentation     | Write Document for each module                                                                                                                                              |
+|0c.     | Testing           | Write test code for each module                                                                                                                                             |
+|0d.      | Example           | Provides code for examples of using DAOS on substrate projects                                                                                                              |
+|0e.      | Comment           | Provide detailed comments for each module                                                                                                                                   |
```

Comment:
Does this refer to inline documentation?

Reply:
> Does this refer to inline documentation?

yes